src/components/cards/CardItem.vue

```diff
@@ -300,9 +300,14 @@ export default {
 	.modal__content {
 		width: 25vw;
 		text-align: center;
-		margin: 4vw 1vw;
+		margin: 20px 20px 60px 20px;
 	}
+
 	.modal__content button{
 		float: right;
+
+		.multiselect {
```

Comment:
.multiselect is not inside the button ;)

Reply:
done